docs/documentation/quartz-3.x/packages/aspnet-core-integration.md

```diff
@@ -37,4 +37,47 @@ public void ConfigureServices(IServiceCollection services)
         options.WaitForJobsToComplete = true;
     });
 }
-```
\ No newline at end of file
+```
+## A practical example of the setup
+
+In the code below you can see a real application of the Quartz package within ASP.NET Core 6 MVC.
+
+To better illustrate the use of the Quartz library, imagine you have a `Program.cs` file that is always created when you choose the MVC architecture, and then imagine a `Jobs` folder where you have all the tasks you want Quartz to perform in the background when you run your web application. 
+
+After that, it's pretty straightforward.
+
+In the `Jobs` folder, you create a class that will perform the tasks you specify.
+The class should extend the `IJob` interface and implement the `Execute` method. 
+
+**Example SendEmailJob.cs configuration**
+```csharp
+public class SendEmailJob : IJob
+{
+    public Task Execute(IJobExecutionContext context)
+        {
+            // Code that sends a periodic email to the user (for example)
+            // Note: This method must always return a value. This is especially important for TriggerListeners that are watching job 
+            return Task.FromResult(true); execution.
+        }
+}        
+```
+After that, you just need to build Quartz trigger in `Program.cs`, which guarantees that the job will run according to the preset interval. 
+
+**Example Program.cs configuration**
+```csharp
+builder.Services.AddQuartz(q =>
+{
+    q.UseMicrosoftDependencyInjectionScopedJobFactory();
+    var jobKey = new JobKey("SendEmailJob"); // Just use the name of your job that you created in the Jobs folder.
+    q.AddJob<SendEmailJob>(opts => opts.WithIdentity(jobKey));
+    
+    q.AddTrigger(opts => opts
+        .ForJob(jobKey)
+        .WithIdentity("SendEmailJob-trigger")
+        .WithCronSchedule("0 * * ? * *")); //This Cron interval can be described as "run every minute"
+});
+builder.Services.AddQuartzHostedService(q => q.WaitForJobsToComplete = true);
+```
+For more information on CronTriggers, you can use the tutorial directly from Quartz - [CronTriggers](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/crontriggers.html).
+
+And for easy generation of Cron intervals using UI you can use - [Free Cron Expression Generator & Explainer](https://www.freeformatter.com/cron-expression-generator-quartz.html).
```

Comment:
I propose to move this to the linked **contriggers** page, if that's ok for @lahma.

Reply:
I think both might be ideal, repetition and maintenance, but people usually miss these. I'd also add http://www.cronmaker.com/ to the mix, maybe link text should contain service name too (freeformatter.com)?